easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -59,10 +59,8 @@
 # not 'easy_install' deliberately, to avoid that pkg installations listed in easy-install.pth get preference
 # '.' is required at the end when using easy_install/pip in unpacked source dir
 EASY_INSTALL_TARGET = "easy_install"
-EASY_INSTALL_INSTALL_CMD = "%(python)s setup.py " + EASY_INSTALL_TARGET + " --prefix=%(prefix)s %(installopts)s %(loc)s"
 PIP_INSTALL_CMD = "pip install --prefix=%(prefix)s %(installopts)s %(loc)s"
 SETUP_PY_INSTALL_CMD = "%(python)s setup.py %(install_target)s --prefix=%(prefix)s %(installopts)s"
-SETUP_PY_DEVELOP_CMD = "%(python)s setup.py develop --prefix=%(prefix)s %(installopts)s"
```

Comment:
In theory, there could be other easyblocks out there that import these constants...

We don't have any in the central easyblocksrepository, but perhaps we should keep these constants for now (and perhaps indefinitely)?

Reply:
While that is possible I'd say this is part of the deprecation we made and hence can be removed. Otherwise we will be stuck with unused constants forever.
As the error will be pretty clear and easy to fix and it will likely be rare anyway I'd remove it.

Reply:
OK, if needed we can always restore them...

Reply:
Ok, so ready to merge? Test reports are above :)